character_structure/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Tak tylko daje znać, że będę trochę podpytywał o python djnago specific rzeczy, bo nie ogarniam - ale po co nam teraz ten admin.py? Czy to na potem?